wmn-data.json

```diff
@@ -5625,6 +5733,18 @@
         "cat" : "XXXPORNXXX",
         "valid" : true
        },
+       {
+        "name" : "Teespring",
+        "uri_check" : "https://commerce.teespring.com/v1/stores?slug={account}",

```

Comment:
Same API question as above. Is there a `pretty_uri` URL right to the profile we can add?